### PR TITLE
Ft rsvp event 153051507

### PR DESCRIPTION
**Pull request by mirr254**
#### What does this PR do?
- Adds dummy data to events model and auth model to make testing easier
#### Description of Task to be completed?
- Have the get endpoint of `api/v1/events` working by default
- Have 1 registered user by default
#### How should this be manually tested?
To test clone the repo and run the app `app.py` 
get default event `/api/v1/events`
#### Screenshots (if appropriate)
dummydat

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 def createApp(conf_name):
 
     app = Flask(__name__)
-    app.config.from_object(app_config[conf_name])
+    #app.config.from_object(app_config[conf_name])
```
Comment (muchai-mercy) [inline app/__init__.py:8]: Do you need this?

File: instance/config.py [hunk 2]
```diff
@@ -3,15 +3,15 @@
 class Config(object):
     #parent configuration class
     DEBUG = False
-    CSRF_ENABLED = True
+    #CSRF_ENABLED = True
```
Comment (muchai-mercy) [inline instance/config.py:6]: Do you need this?

File: app/__init__.py (hunk 1 above)
Comment (mirr254) [reply app/__init__.py:8]: For the time being, no

File: instance/config.py (hunk 2 above)
Comment (mirr254) [reply instance/config.py:6]: No. Should I remove them?

Comment (muchai-mercy) [reply instance/config.py:6]: Yes. It is good practice to clean up commented out code that you are not using

**Pull request merged**